main/test/flix/CompilerSuite.scala

```diff
@@ -44,6 +44,51 @@ class CompilerSuite extends FlixSuite(incremental = true) {
   mkTest("main/test/flix/Test.Exp.Binary.Logic.flix")
   mkTest("main/test/flix/Test.Exp.Binary.Spaceship.flix")
 
+  //
+  // Block.
+  //
+  mkTest("main/test/flix/Test.Exp.Block.flix")
+
+  //
+  // Cast.
+  //
+  mkTest("main/test/flix/Test.Exp.Cast.flix")
+
+  //
+  // Char.
+  //
+  mkTest("main/test/flix/Test.Exp.Char.flix")
+
+  //
+  // Choose.
+  //
+  mkTest("main/test/flix/Test.Exp.Choose.flix")
+  mkTest("main/test/flix/Test.Exp.ChooseStar.flix")
+
+  //
+  // Concurrency.
+  //
+  mkTest("main/test/flix/Test.Exp.Concurrency.Buffered.flix")
+  mkTest("main/test/flix/Test.Exp.Concurrency.NewChannel.flix")
+  mkTest("main/test/flix/Test.Exp.Concurrency.Unbuffered.flix")
+  mkTest("main/test/flix/Test.Exp.Concurrency.Spawn.flix")
+  mkTest("main/test/flix/Test.Exp.Concurrency.Select.flix")(Options.DefaultTest.copy(xallowredundancies = true))
```

Comment:
Is this correct?

Reply:
I forget this specific case, but ideally we are trying to avoid these flags.

What happens if you drop allow redundances? And if it gives errors, can you fix those?